unlock-app/src/hooks/useAccount.ts

```diff
@@ -9,6 +9,27 @@ import {
   reEncryptPrivateKey,
 } from '../utils/accounts'
 import { ToastHelper } from '~/components/helpers/toast.helper'
+import { useWeb3Service } from '~/utils/withWeb3Service'
+import { useQuery } from '@tanstack/react-query'
+
+export const useAccountBalance = ({
```

Comment:
I think we may already have a version of this somewhere?

Reply:
replaced with `useAccountBalance`